src/asserters/script.ts

```diff
@@ -0,0 +1,18 @@
+import AsserterBase from './base'
+import {exec} from '../utils'
+
+export class RunCommandsAsserter extends AsserterBase {
+  protected async uniqWork() {
+    if (!this.assertion.commands) {
+      throw new Error('No commands were provided in the sequence.')
+    }
+
+    const commands: string[] = Array.isArray(this.assertion.commands) ?
+      this.assertion.commands :
+      [this.assertion.commands]
+
+    for (const cmd of commands) {
+      await exec(cmd, {cwd: this.workingDir})
```

Comment:
If a command fails here just it will throw the error from `child_process` with both stdout/stderr and make leif stop.